react plugin not mentioned in package.json

react plugin not mentioned in package.json

Yes you are right. I even get the error message when trying to run:

```
jakoblind@MBPsomthorJakob ~/D/empty-project (15)> npx eslint src/*
Failed to load plugin 'react' declared in '.eslintrc.json': Cannot find module 'eslint-plugin-react'
Require stack:
- /Users/jakoblind/Downloads/empty-project (15)/__placeholder__.js
```

Thanks for reporting. This should be fixed!

--- a/src/configurator/common-config/linting.js
+++ b/src/configurator/common-config/linting.js
@@ -51,7 +51,13 @@ function createEsLintRc(answers = { moduleType: 'esm' }) {
 
 const eslint = {
   group: 'Linting',
-  devDependencies: configItems => ['eslint'],
+  devDependencies: configItems => {
+    const isReact = _.includes(configItems, 'React');
+    return _.concat(
+      ['eslint'],
+      isReact ? ['eslint-plugin-react'] : []
+    );
+  },
   files: configItems => {
     const isReact = _.includes(configItems, 'React');
     const isVue = _.includes(configItems, 'Vue');

